src/chigame/api/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path("games/", views.GameListView.as_view(), name="game-list"),
+    path("games/<int:pk>/", views.GameDetailView.as_view(), name="game-detail"),
```

Comment:
Nice job actually providing names. 
Consider adding `api-` in front of your names here, since these names are used outside the scope of just your app, so this could get confused with a `games` app URL.

Instead something like `api-game-list` is unmistakably the API's url.

Reply:
added the "api-" prefix before the names of the API views